### PR TITLE
Rephrase descriptions to < 135 characters

### PR DESCRIPTION
**Pull request by frankharkins**
### Summary

The new ecosystem page will have a 135 character limit for descriptions. This PR rephrases any long descriptions while trying to preserve their meaning.

### CODE REVIEW

File: ecosystem/resources/members/QoptKIT.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name = "QoptKIT"
 url = "https://github.com/SOQCSAdmin/QoptKIT"
-description = "Optical circuits in Qiskit. Translate Qiskit circuits to quantum-optical circuits made of phase shifters and beamsplitters, simulate the circuit, then translate the outcome back to a qubit encoding."
+description = "Translate Qiskit circuits to quantum-optical circuits, simulate them, then translate results back to qubit encodings."
```
Comment (beckykd) [inline ecosystem/resources/members/QoptKIT.toml:3]:
```suggestion
description = "Translate Qiskit circuits to quantum-optical circuits, simulate them, then translate the results back to qubit encodings."
```

File: ecosystem/resources/members/qiskit-superstaq.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 name = "qiskit-superstaq"
 url = "https://github.com/Infleqtion/client-superstaq/tree/main/qiskit-superstaq"
-description = "This package is used to access SuperstaQ via a Web API through Qiskit. Qiskit programmers can take advantage of the applications, pulse level optimizations, and write-once-target-all features of SuperstaQ with this package."
+description = "Access SuperstaQ via a Web API through Qiskit. Take advantage of pulse-level optimizations, write-once-target-all features, and more."
```
Comment (beckykd) [inline ecosystem/resources/members/qiskit-superstaq.toml:3]:
```suggestion
description = "Access SuperstaQ by using a Web API through Qiskit. Use pulse-level optimizations, write-once-target-all features, and more."
```

File: ecosystem/resources/members/qiskit-symb.toml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 name = "qiskit-symb"
 url = "https://github.com/SimoneGasperini/qiskit-symb"
-description = "Easy-to-use Python package designed to enable symbolic quantum computation in Qiskit. It provides the basic tools for the symbolic evaluation of statevectors, density matrices, and unitary operators directly created from parametric Qiskit quantum circuits. The implementation is based on the Sympy library as backend for symbolic expressions manipulation."
+description = "Symbolic evaluation of statevectors, density matrices, and unitary operators created from parameterized circuits, in Qiskit."
```
Comment (beckykd) [inline ecosystem/resources/members/qiskit-symb.toml:3]:
```suggestion
description = "Symbolic evaluation of statevectors, density matrices, and unitary operators created from parameterized circuits in Qiskit."
```

File: ecosystem/resources/members/vqls-prototype.toml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 name = "Variational Quantum Linear Solver Prototype"
 url = "https://github.com/QuantumApplicationLab/vqls-prototype"
-description = "The Variational Quantum Linear Solver (VQLS) uses an optimization approach to solve linear systems of equations. The vqls-prototype allows to easily setup and deploy a VQLS instance on different backends through the use of qiskit primitives and the runtime library."
+description = "VQLS uses an optimization approach to solve linear systems of equations. Set up and deploy a VQLS instance on different backends."
```
Comment (beckykd) [inline ecosystem/resources/members/vqls-prototype.toml:3]:
```suggestion
description = "VQLS uses an optimization approach to solve systems of linear equations. Set up and deploy VQLS instances on different backends."
```

Comment (beckykd) [reply ecosystem/resources/members/vqls-prototype.toml:3]: "linear systems of equations" isn't a thing; right? I know a "system of linear equations" is a thing.

**Review by beckykd — APPROVED** [review]

**Review by Eric-Arellano — APPROVED** [review]
I didn't closely review - approving so that you can merge, thanks to Becky's review

File: ecosystem/resources/members/qiskit-superstaq.toml (hunk 2 above)
Comment (frankharkins) [reply ecosystem/resources/members/qiskit-superstaq.toml:3]: Not sure about "by using a web API", can't say why but it sounds unusual to me. What about this?

```suggestion
description = "Access the SuperstaQ web API through Qiskit for applications, pulse-level optimizations, and write-once-target-all features."
```

File: ecosystem/resources/members/vqls-prototype.toml (hunk 4 above)
Comment (frankharkins) [reply ecosystem/resources/members/vqls-prototype.toml:3]: I think you're right

File: ecosystem/resources/members/qiskit-superstaq.toml (hunk 2 above)
Comment (beckykd) [reply ecosystem/resources/members/qiskit-superstaq.toml]: LGTM

Comment (beckykd) [reply ecosystem/resources/members/qiskit-superstaq.toml]: "by using" does sound a little strange.  We're not supposed to use "via" and we'd already used "through", which is why I ended up there.  But your rewrite is fine with me

**Pull request merged**